### PR TITLE
Bugfix/42 rsvp change

### PR DESCRIPTION
**Pull request by anhtin**
This PR addresses a small part of #42: allowing users to change their replies (i.e. drop an event after they've previously chosen to join).

I'm marking this PR as a draft as we will need to discuss the interface of communicating replies. The solution in this PR is a suggestion for how we can support both operations with minimal change to the API.

### CODE REVIEW

**Review by martinothamar — COMMENTED** [review]
LGTM, I like the 'refresh' improvement, really clean 🥇

**Comment by anhtin** [comment]
Not my original idea, but I think so too! 👍

**Review by LERoise — APPROVED** [review]

**Comment by duytantran** [comment]
Minor changes. Tested with api.

File: src/components/inputs/RsvpButton/RsvpButton.tsx [hunk 1]
```diff
@@ -1,48 +1,75 @@
...
+  event,
+  onSuccess,
+  onError,
+}: RsvpButtonProps & RouteComponentProps) {
   const account = AuthProviderInstance.account;
 
-  const [hasJoinedEvent, setHasJoinedEvent] = useState(account &&
-    (
-      event.participants
+  const isOwner = account && event.owner?.id === account.localAccountId;
```
Comment (anhtin) [inline src/components/inputs/RsvpButton/RsvpButton.tsx:22]: Maybe move this further down to where it's being used

**Review by anhtin — COMMENTED** [review]
👍

**Review by LERoise — APPROVED** [review]

**Review by duytantran — APPROVED** [review]

**Pull request merged**